src/requirements/data/index.ts

```diff
@@ -130,30 +140,45 @@ const json: RequirementsJson = {
     }
   },
   minor: {
-    DBME: {
-      name: 'Dyson Business Minor for Engineers',
```

Comment:
Are we sure that the business minor is the same for everyone outside of engineering?

Reply:
I think there's three diff business minors (Business (which i also added), this one, and one for Life Sciences Majors) that are all different. So I just went with the name that the Johnson school gave each one